app/networkapi/utility/templatetags/adminsortable_tags_custom.py

```diff
@@ -46,6 +46,8 @@ def render_object_rep(
 ):
     context.update({'object': obj, 'forloop': forloop})
     tmpl = template.loader.get_template(
-        'features/adminsortable_objects_custom.html'
+        '{app_name}/adminsortable_objects_custom.html'.format(
```

Comment:
this is my hack to make this template tag app-agnostic. This had to be done because what we really wanted was to have a templatetag for each app that uses the custom object template for it's corresponding app but, is named the same as all the other apps' custom object templatetags. Since that is impossible (you cannot have multiple templatetags all with the same name that do different things), I use the app_label (automatically defined in the meta class of a model) to determine the app that this object should be rendered for and use that to determine where the custom object template lives. The consequences of this are simple - whenever you add sorting to another app, all you need to do is:
- Add the integer field to the model and update the ordering in the meta class
- Inherit from the correct classes for your admin model and regular model
- set the `sortable_change_list_template` to the `list_template` in the shared set of templates
- create your object representation template in the exact path: `app/networkapi/templates/<name of the app>/adminsortable_objects_custom.html`